internal/cloud/realm/app.go

```diff
@@ -109,7 +109,7 @@ func (c *client) FindApps(filter AppFilter) ([]App, error) {
 
 	var filtered = make([]App, 0, len(apps))
 	for _, app := range apps {
-		if strings.HasPrefix(app.ClientAppID, filter.App) {
+		if strings.HasPrefix(app.ClientAppID, strings.ToLower(filter.App)) {
```

Comment:
Noticed the ClientAppID would be all lowercased which would cause name not to be matched